saleor/graphql/app/resolvers.py

```diff
@@ -19,3 +23,10 @@ def resolve_access_token(info, root, **_kwargs):
     if user.is_anonymous:
         return None
     return create_access_token_for_app(root, user)
+
+
+@permission_required(AppPermission.MANAGE_APPS)
```

Comment:
Other app resolvers have the decorator defined at the schema.py level. I think it would be more consistent to keep it there. I think a dedicated function `_resolve_app` is not needed since it's a one-line resolver.

Reply:
I did that because I wanted to wrap only this part with permission, if we provide the token app we should return `app` object without checking the permission.

Reply:
OK, that makes sense 👍